README.md

```diff
@@ -39,26 +39,20 @@ The ROS package can be installed and invoked by the following commands.
 ### Installation
 
 ```
-git clone https://github.com/jk-ethz/franka_lock_unlock
-catkin build franka_lock_unlock
+git clone https://github.com/jk-ethz/franka_lock_unlock.git
+colcon build
 ```
 
 ### Simple Usage
 
-```
-rosrun franka_lock_unlock __init__.py <PARAMS>
+```sh
+ros2 run franka_lock_unlock franka_lock_unlock.py <PARAMS>
 ```
 
 ### Advanced Usage
 
-If you want to launch the `franka_control` node while unlocking the joints, ensure to add the `respawn=true` parameter to the node tag, such that the control node retries to connect to the robot until all joints have been fully unlocked and the FCI has been activated.
-
-<pre>
-&lt;node name="franka_control" pkg="franka_control" type="franka_control_node" output="screen" <b>respawn="true"</b> /&gt;
-</pre>
-
```

Comment:
Is everything working fine without this respawning if you launch Franka and this package at the same time?